test/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+        "strict": false,
+        "allowJs": true,
+        "checkJs": false
+    },
+    "include": ["../node_modules/@types/mocha/index.d.ts", "./mongodb-mocha.d.ts",]
```

Comment:
I don't think the trailing comma is meant to be here.  In fact - doesn't JSON not allow trailing commas at all?  I'm surprised VSCode didn't complain about this

Reply:
tsconfig files can be written in JSONC, comments and trailling commas are allowed. I don't think I need this file anymore anyway though